functions/go/apply-setters/generated/docs.go

```diff
@@ -83,7 +81,7 @@ Invoke the function:
 
 Alternatively, setter values can be passed as key-value pairs in the CLI
 
-  $ kpt fn eval --image gcr.io/kpt-fn/apply-setters:unstable -- image=ubuntu replicas=3
+  $ kpt fn eval --image gcr.io/kpt-fn/apply-setters:unstable -- tag=1.16.2 nginx-replicas=3
```

Comment:
qq: do we need to change this example?

Reply:
(I had no intention of making this change btw. It probably was made when running `make`)